inference-engine/tests/functional/inference_engine/CMakeLists.txt

```diff
@@ -5,25 +5,40 @@
 
 set(TARGET_NAME ieFuncTests)
 
+set(INCLUDES ${IE_MAIN_SOURCE_DIR}/src/inference_engine)
+set(LINK_LIBRARIES
+    gmock
+    funcTestUtils
+    ngraphFunctions
+    inference_engine_transformations
+)
+set(DEPENDENCIES
+    mock_engine
+    inference_engine_ir_reader
+    inference_engine_ir_v7_reader
+    template_extension
+)
+
+if (NGRAPH_ONNX_IMPORT_ENABLE)
+    set(INCLUDES ${INCLUDES} ${OpenVINO_MAIN_SOURCE_DIR}/docs/onnx_custom_op)
+    set(LINK_LIBRARIES ${LINK_LIBRARIES} onnx_importer onnx_custom_op)
+    set(DEPENDENCIES ${DEPENDENCIES} onnx_importer onnx_custom_op)
```

Comment:
I think `onnx_custom_op` is already a dependency for `template_extension`. Do we need to add manually to the lists of `LINK_LIBRARIES`, `DEPENDENCIES`? I suppose no.

Reply:
onnx_custom_op is not a dependency for template_extension, but I can remove onnx_importer dep here.

Reply:
ok, let's remove onnx_importer